infra/cifuzz/cifuzz.py

```diff
@@ -185,10 +192,13 @@ def run_fuzzers(fuzz_seconds, workspace):
     return False, False
   fuzz_seconds_per_target = fuzz_seconds // len(fuzzer_paths)
 
+  ossfuzz_build_path = download_ossfuzz_build(project_name, out_dir)
```

Comment:
We usually call it "oss_fuzz" in code. 
In another CL can you change the unrelated code that calls it "ossfuzz"?